templates/partials/page-footer.hbs

```diff
@@ -1,4 +1,4 @@
-<footer role="contentinfo">
+<footer role="contentinfo" class="{{class}}">
```

Comment:
Could we use a more specific name than `class` for the key - maybe `footerModifierClass` or similar?

Also, maybe make the class attribute appear only when that key is set, so something like:

`{{#if footerModifierClass}} class="{{footerModifierClass}}"{{/if}}`

Reply:
I changed the key name as you suggested, thanks! About `#if` it currently works as you mentioned: the class attribute only appear when the key is set with a value, otherwise it doesn't appear. I tested with an undefined key, as well as a key with an empty value.